src/Avalonia.Base/Input/DragEventArgs.cs

```diff
@@ -32,7 +33,9 @@ public Point GetPosition(Visual relativeTo)
             return point;
         }
 
-        internal DragEventArgs(RoutedEvent<DragEventArgs> routedEvent, IDataObject data, Interactive target, Point targetLocation, KeyModifiers keyModifiers)
+        [Unstable]
+        [Obsolete("This constructor may be removed in 12.0. Consider replacing it with RawDragEvent.")]
```

Comment:
Aren't the raw events also considered unstable? I thought that the alternative was to use the headless platform for unit tests, and raise custom pointer events at all outside unit tests.

Reply:
Added DragDrop method to the IHeadlessWindow, updated obsolete message. Removed mentioning of raw events.